content/open-source/sigstore/policy-controller/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Policy Controller"
+lead: ""
+type: "article"
+date: 2023-02-21T13:05:12+08:12
+lastmod: 2023-02-21T13:05:12+08:12
+draft: false
+menu:
+  docs:
+    parent: "sigstore"
+images: []
+---
+
+A Kubernetes admission controller can be used to validate signatures and attestations on container images as well as enforce policies using cue or rego languages.
```

Comment:
Do we need the "can be" language, it seems tentative to me

```suggestion
A Kubernetes admission controller used to validate signatures and attestations on container images as well as enforce policies using CUE or Rego languages.
```

Reply:
More direct, I like it